.gitignore

```diff
@@ -45,3 +45,6 @@ yarn-error.log*
 .idea/
 .exrc
 .envrc
+
+yarn.lock
+.gitpod.yml
```

Comment:
Do we need this? :eyes:  If not, please remove it

Reply:
Whoops, very silly mistake.
Fixed it.